### PR TITLE
Update rollbar version and add scrub fields and headers so we don't l…

### PR DESCRIPTION
**Pull request by Adovenmuehle**
…eak sensitive information. Also bumped gemspec version.

### CODE REVIEW

**Comment by Adovenmuehle** [comment]
Security Incident number 6.

**Comment by rmcafee** [comment]
:+1:

**Comment by Adovenmuehle** [comment]
Build is going to fail until we release the latest gem to rubygems

File: lib/vault-tools.rb [hunk 1]
```diff
@@ -9,6 +9,12 @@
 Rollbar.configure do |config|
   config.environment = ENV['RACK_ENV'] || ENV['RAILS_ENV'] || ENV['APP_ENV'] || ENV['ROLLBAR_ENV'] || 'unassigned'
   config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
+  config.scrub_headers |= ["Authorization", "Cookie", "Set-Cookie", "X_CSRF_TOKEN", "X-CSRF-Token", "HTTP_X_CSRF_TOKEN"]
```
Comment (kennyp) [inline lib/vault-tools.rb]: This overrides the defaults that come with the latest version of Rollbar. Are we missing any in their list?

**Comment by kennyp** [comment]
Looks good.

File: lib/vault-tools.rb (hunk 1 above)
Comment (Adovenmuehle) [inline lib/vault-tools.rb]: Shouldn't override with the |=, right?

Comment (kennyp) [inline lib/vault-tools.rb]: It won't override, but don't you want to just add to their list? Also I think you want `||=` instead of `|=`.

**Comment by rmcafee** [comment]
:+1:

**Pull request merged**